src/AuthenticatedAPIClient/authInterface.js

```diff
@@ -37,11 +34,11 @@ export default function applyAuthInterface(httpClient, authConfig) {
   httpClient.handleRefreshAccessTokenFailure = authConfig.handleRefreshAccessTokenFailure;
   httpClient.loggingService = authConfig.loggingService;
 
-  /**
-   * We will not try to refresh an expired access token before
-   * making requests to these auth-related URLs.
-   */
-  httpClient.authUrls = [httpClient.refreshAccessTokenEndpoint];
```

Comment:
Internally, AccessToken uses its own instance of axios that does not have our auth interceptors on it. Assuming the jwt token refresh endpoint is the only url that should be exempt, we no longer need to check or handle exceptions.